web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -212,15 +211,15 @@
       {#if $slideshowState !== SlideshowState.None && $slideshowLook === SlideshowLook.BlurredBackground}
         <img
           src={assetFileUrl}
-          alt={$getAltText(asset)}
```

Comment:
We probably need to have some default alt text, even from just the timeline asset.

Reply:
This image is for the background blur image - as such, it is considered 'decorative'. https://www.w3.org/WAI/tutorials/images/decorative/
The guide recommends a null alt property, since its more widely supported. Otherwise, using role="presentation" is perhaps has a little more clarity/readability from code POV.

Reply:
Ah, ok!